test/sqlancer/dbms/TestSQLite3.java

```diff
@@ -12,7 +12,7 @@
     public void testSqlite() {
         // run with one thread due to multithreading issues, see https://github.com/sqlancer/sqlancer/pull/45
         assertEquals(0, Main.executeMain(new String[] { "--random-seed", "0", "--timeout-seconds", TestConfig.SECONDS,
-                "--num-threads", "1", "--num-queries", "0", "sqlite3" }));
+                "--num-threads", "1", "--num-queries", "100", "sqlite3" }));
```

Comment:
Could you please set `--num-queries` back to `0`? NoREC can still find bugs in the SQLite version included in the latest JDBC driver, so this test might fail sometimes.

Reply:
Accidentally added changes that I used during debug. Reverted it.